client/utils/lst.c

```diff
@@ -0,0 +1,532 @@
+//
+//  lst.c
```

Comment:
What is the purpose of this new list implementation? And where it is from? We already have list implementation in ulist.h

Reply:
Agree, better to avoid creating different implementations of some basic algorithms. Please use ulist or explain us why that implementation does not fit your pusposes.